src/main/resources/static/js/deleteSubject.js

```diff
@@ -49,14 +45,13 @@ document.addEventListener('DOMContentLoaded', function () {
                 confirmationModal.hide();
                 if (response.ok) {
                     selectedButton.closest('tr').remove();
-                    const successAlert = document.getElementById('success-alert');
-                    successAlert.style.display = 'block';
-                    setTimeout(function () {
-                        successAlert.style.display = 'none';
-                    }, 3000); 
+                    console.log("Subject successfully deleted.");
+
+                        window.location.reload();
+
                 } else {
                     return response.text().then(text => {
```

Comment:
what if there is a server error?

Reply:
do you want me to change catch error to: 
.catch(error => {
                confirmationModal.hide();
                console.error('Server error:', error);
                alertModalBody.textContent = 'An error occurred. Please try again later.';
                alertModal.show();
            }